sparse/usr/bin/droid/android-links.sh

```diff
@@ -1,14 +1,24 @@
 #!/usr/bin/env bash
 
-MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)
+MNT=$(grep "Where=" /lib/systemd/system/android.mount | cut -d'=' -f 2)/media/0
+
+[[ "$(readlink -- /home/nemo/Pictures/Android)" == "/android/Pictures" ]] && unlink /home/nemo/Pictures/Android || echo "clean"
```

Comment:
Please turn this check into an if-block, and unlink all in that block. It's clearer that way, and we won't try to unlink Pictures/Android twice as we do now :)